packages/data/package.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "@wordpress/data",
+	"version": "0.0.1",
+	"description": "Data Redux module for WordPress",
```

Comment:
The documentation clearly states:

>The data module is built upon and shares many of the same core principles of [Redux](https://redux.js.org/), but shouldn't be mistaken as merely _Redux for WordPress_

Is this not in contradiction?

Reply:
It is, I will update.